MOCC/src/configurations/NewPyTango/HealthMonitorDS.py

```diff
@@ -0,0 +1,39 @@
+from flask import Flask, jsonify
+from flasgger import Swagger
+
+
+app = Flask(__name__)
+app.config['SWAGGER'] = {
+    'title': 'Health Monitor API',
+    'uiversion': 2
+}
+Swagger(app)
```

Comment:
I am trying to understand flassger.
From what I've seen so far, it allows you to make documentations like the petstore example given by them.

Could you please give an example of the kind of functions I should call with flassger?
Where and why should flasgger be used?